hack/test-prerequisites.go

```diff
@@ -36,12 +36,12 @@ func main() {
 			if err != nil {
 				log.Fatalf("Unable to read %s: %v", path, err)
 			}
-			var crd v1beta1.CustomResourceDefinition
+			var crd v1.CustomResourceDefinition
 			if err := yaml.Unmarshal(data, &crd); err != nil {
```

Comment:
You don't check that the API version matches? That's not going to make you happy when it doesn't.

Reply:
ok, test only.

Reply:
I don't think we do that in tests. In actual flow non-v1 CRDs would be treated as "unstructered".

@wking @LalatenduMohanty any advice here? I suppose we should be tracking how many manifests are being applied by unstructured handler and attempt to minimize that number. Probably we should be exporting a new metric?

Reply:
My long-term goal is to completely drop the unstructured handler, and fail folks in CI if they add a new manifest type that we don't support.  Logging in the unstructured handler that we can look for in gathered CI assets would help get us there.

Reply:
> if they add a new manifest type that we don't support. Logging in the unstructured handler that we can look for in gathered CI assets would help get us there.

+1